src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1758,8 +1758,10 @@ private List<CompletionResult> GetResultForAttributeArgument(CompletionContext c
                 List<CompletionResult> result = new List<CompletionResult>();
                 foreach (PropertyInfo pro in propertyInfos)
                 {
-                    //Ignore TypeId (all attributes inherit it)
-                    if (pro.Name != "TypeId" && (pro.Name.StartsWith(argName, StringComparison.OrdinalIgnoreCase)))
```

Comment:
why did the `!= "TypeId` go away?

Reply:
The property `TypeId` is a read-only property. This change adds two more read-only properties to `ParameterAttribute`: `ExperimentName` and `ExperimentAction`. They shouldn't show up in the tab completion of Attribute properties because you cannot set values for them. So we should just ignore all getter properties.